template/datepicker/day.html

```diff
@@ -1,9 +1,9 @@
 <table role="grid" aria-labelledby="{{::uniqueId}}-title" aria-activedescendant="{{activeDateId}}">
   <thead>
     <tr>
-      <th><button type="button" class="btn btn-default btn-sm pull-left uib-left" ng-click="move(-1)" tabindex="-1"><i class="glyphicon glyphicon-chevron-left"></i></button></th>
+      <th><button type="button" aria-label="Previous Month" class="btn btn-default btn-sm pull-left uib-left" ng-click="move(-1)" tabindex="-1"><i class="glyphicon glyphicon-chevron-left"></i></button></th>
```

Comment:
After looking into it a bit more, I followed the pattern currently being used for the carousel's forward and back buttons, and used `aria-hidden=true` on the glyphicon, along with a `<span class="sr-only">screen-reader text</span>`.  It doesn't fix the issue with localization (which the carousel also suffers from), but widespread localization of strings is probably big enough for a separate issue.  Thoughts?